tools/docker/fabric-all-in-one/run-fabric-network.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 set -e
 
-# TODO: make the verion numbers here injectable to the script
-/bootstrap.sh 1.4.8 1.4.8 -b -s
+/bootstrap.sh ${FABRIC_VERSION} 1.4.8 -b -s
```

Comment:
Same two asks as above: 1) typo of default CA version, 2) we can have an ENV variable for the CA version as well (and it can be optional with the default value taken from the bootstrap script of Fabric itself, 1.4.9)

Reply:
Incorporated the changes. Please have a re-look.